airdrop/simple-token/tests/functions/mint_to.rs

```diff
@@ -0,0 +1,145 @@
+use crate::utils::{
+    abi_calls::{constructor, mint_to},
+    test_helpers::setup,
+    Identity,
+};
+use fuels::{signers::Signer, tx::AssetId};
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn mints_to_one_wallet() {
+        let (deployer, _) = setup().await;
+
+        let identity = Identity::Address(deployer.wallet.address().into());
+        constructor(identity.clone(), &deployer.simple_token, 100).await;
+
+        assert_eq!(
+            deployer
+                .wallet
+                .get_asset_balance(&AssetId::new(*deployer.asset_id))
+                .await
+                .unwrap(),
+            0
+        );
+
+        mint_to(10, &deployer.simple_token, identity.clone()).await;
+
+        assert_eq!(
+            deployer
+                .wallet
+                .get_asset_balance(&AssetId::new(*deployer.asset_id))
+                .await
+                .unwrap(),
+            10
+        );
+    }
+
+    #[tokio::test]
+    async fn mints_to_multiple_wallets() {
+        let (deployer, wallet2) = setup().await;
+
+        let identity = Identity::Address(deployer.wallet.address().into());
+        constructor(identity.clone(), &deployer.simple_token, 100).await;
+
+        assert_eq!(
+            deployer
+                .wallet
+                .get_asset_balance(&AssetId::new(*deployer.asset_id))
+                .await
+                .unwrap(),
+            0
+        );
+        mint_to(10, &deployer.simple_token, identity.clone()).await;
+        assert_eq!(
+            deployer
+                .wallet
+                .get_asset_balance(&AssetId::new(*deployer.asset_id))
+                .await
+                .unwrap(),
+            10
+        );
+
+        let identity2 = Identity::Address(wallet2.wallet.address().into());
```

Comment:
I would try to keep all the declarations in 1 spot at the top so that the reader does not have to browse through the test to see where something is declared